modules/core/04-channel/v2/types/commitment.go

```diff
@@ -6,6 +6,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+var FailedAcknowledgementCommitment = sha256.Sum256([]byte{byte(2), byte(0)})
```

Comment:
I've left this as a var for now but can change to a string const if preferred.

Need feedback from @srdtrk if this constant err ack hash is acceptable